lib/build_manager.py

```diff
@@ -47,6 +49,48 @@ def __call__(self):
 
         self.build()
 
+    def _build_packages(self, distro, packages):
+        """
+        Build packages
+
+        Args:
+            distro (Distribution): Linux distribution
+            packages ([str]): packages names
+        """
+
+        # create package builder based on distro
+        if distro.lsb_name == "CentOS":
+            mock_config_file = CONF.get('build_packages').get('mock_config').get(
+                distro.lsb_name).get(distro.version)
+            package_builder = mockbuilder.Mock(mock_config_file)
```

Comment:
There is a 1-N relationship between the a package builder and a distro. Handling this logic in the distro class or a third class are the possible solutions. As the number of distros we'll support is small and the logic to infer the package builder from the distro is very simple, IMHO keeping this logic in a third class (build manager) is clearer.